docs/en/docs/tutorial/index.md

```diff
@@ -2,15 +2,13 @@
 
 This tutorial shows you how to use **FastAPI** with most of its features, step by step.
 
-Each section gradually builds on the previous ones, but it's structured to separate topics, so that you can go directly to any specific one to solve your specific API needs.
+Each section gradually builds on the previous ones. Each section is a separate topic, so that you can go directly to any specific topic to solve your specific API need.
 
-It is also built to work as a future reference.
-
-So you can come back and see exactly what you need.
+The tutorial is also built to work as a future reference, so you can come back and see exactly what you need.
 
 ## Run the code
 
-All the code blocks can be copied and used directly (they are actually tested Python files).
+All code blocks can be copied and used directly (they are tested Python files).
```

Comment:
What do you think is wrong with the original version?

Reply:
If extra words don't add anything to a sentence it is best to remove them.  The text can be read faster and the writing is crisper.  I just picked a guide on the topic at random and "actually" shows up on the list of words to cut   🙂: https://owl.purdue.edu/owl/general_writing/academic_writing/conciseness/eliminating_words.html